examples/3d/skybox.rs

```diff
@@ -37,7 +40,8 @@ const CUBEMAPS: &[(&str, CompressedImageFormats)] = &[
 
 fn main() {
     App::new()
-        .add_plugins(DefaultPlugins)
+        .add_plugins((DefaultPlugins, TemporalAntiAliasPlugin))
+        .insert_resource(Msaa::Off)
```

Comment:
For a user quickly looking at examples for how to do things it's easy to forget why one shouldn't use msaa in this context.
I've forgotten it myself but I think I've read somewhere (discord? other example?) that it's not compatible with _something_, TAA perhaps?

Warrents a comment imo